pwndbg/dbg/__init__.py

```diff
@@ -251,6 +254,39 @@ def remove(self) -> None:
         raise NotImplementedError()
 
 
+class EventType(Enum):
+    """
+    Events that can be listened for and reacted to in a debugger.
+
+    The events types listed here are defined as follows:
+        - `START`: This event is fired some time between the creation of or
+          attachment to the process to be debugged, and the start of its
+          execution.
+        - `STOP`: This event is fired after execution of the process has been
+          suspended, but before control is returned to the user for interactive
+          debugging.
+        - `EXIT`: This event is fired after the process being debugged has been
+          detached from or has finished executing.
+        - `MEMORY_CHANGED`: This event is fired when the user interactively makes
+          changes to the memory of the process being debugged.
+        - `REGISTER_CHANGED`: Like `MEMORY_CHANGED`, but for registers.
+        - `CONTINUE`: This event is fired after the user has requested for
+          process execution to continue after it had been previously suspended.
+        - `NEW_MODULE`: This event is fired when a new application module has
+          been encountered by the debugger. This usually happens when a new
+          application module is into the memory space of the process being
```

Comment:
```suggestion
          application module is loaded into the memory space of the process being
```

Also: why 'module' and not 'objfile'? Is that naming from llvm? Maybe we should say `new module (objfile)` in the description here

Reply:
Yeap, that's what LLDB calls them. Generally, the Debugger-agnostic API sticks with LLDB whenever GDB and LLDB disagree on what to call something. But yeah, the clarification there is a good idea.

Reply:
I've added "In GDB terminology, these are called `objfile`s." to the end of line for `NEW_MODULE`. I think that should clear things up?